chalicelib_fourfront/checks/helpers/wfr_utils.py

```diff
@@ -540,9 +352,9 @@ def get_wfr_out(emb_file, wfr_name, key=None, all_wfrs=None, versions=None,
 
     error_at_failed_runs = error_threshold
     # you should provide key or all_wfrs
-    assert key or all_wfrs
-    if wfr_name not in workflow_details:
-        assert wfr_name in workflow_details
+    assert key
+    workflow_details = get_workflow_details(key)
+    assert wfr_name in workflow_details
```

Comment:
Ran into an error at this point in a check running--it looks like `workflow_details` is just `{}`. I will keep exploring!

Reply:
This makes sense as the properties that were added to the Workflow and ExperimentType items are not in webdev and I guess the version of fourfront there is likely prior to the schema changes so they can't be added.  Note that for testing particular workflows you can pass in this info as a parameter but that kind of defeats some of the purpose of testing.  I will consider options for making this backward compatible so it can be used on webdev without errors.

Reply:
Oh, of course!